### PR TITLE
New alert style

### PR DESCRIPTION
**Pull request by mxsnyder**
Mostly style changes. This is a quick one.

### CODE REVIEW

File: js/apps/events-page/dataServices/eventsService.js [hunk 1]
```diff
@@ -4,6 +4,9 @@
 	const eventsService = (CONSTANTS, $http, $q) => {
 		const isEventOnDate = (eventItem, eventDate) => {
 			const eventItemStartDateLocaleString = (new Date(eventItem.EventStart)).toLocaleDateString();
+
+			console.log(eventItemStartDateLocaleString, eventDate);
```
Comment (martypowell) [inline js/apps/events-page/dataServices/eventsService.js]: remove?

File: stylesheets/partials/features/_header-alert.scss [hunk 2]
```diff
@@ -1,30 +1,26 @@
...
+	position: relative;
 	top: -1px;
 	
 	&.emergency {
 		background-color: $emergency-orange;
 		color: white;
 
 		p {
-			& > span i {
+			& > span.message i {
```
Comment (martypowell) [inline stylesheets/partials/features/_header-alert.scss]: does this need span?

File: stylesheets/partials/features/_header-alert.scss [hunk 3]
```diff
@@ -36,40 +32,38 @@
 		margin: 0;
 		margin-top: -4px;
 
-		& > span i {
+		& > span.message i {
```
Comment (martypowell) [inline stylesheets/partials/features/_header-alert.scss]: does this need span?

File: js/apps/events-page/dataServices/eventsService.js (hunk 1 above)
Comment (mxsnyder) [reply js/apps/events-page/dataServices/eventsService.js]: Done.

File: stylesheets/partials/features/_header-alert.scss (hunk 2 above)
Comment (mxsnyder) [reply stylesheets/partials/features/_header-alert.scss]: Done.

File: stylesheets/partials/features/_header-alert.scss (hunk 3 above)
Comment (mxsnyder) [reply stylesheets/partials/features/_header-alert.scss]: Done.

**Comment by mxsnyder** [comment]
Ready for another look.

**Pull request merged**